docs/features/vpn.rst

```diff
@@ -82,13 +82,37 @@ To enable L2TP offloading on the supornodes as well, it is recommended to study
 the fastd documentation section pertaining to the `offload configuration option
 <https://fastd.readthedocs.io/en/stable/manual/config.html#option-offload>`_.
 
+
+Changes necessary in your Servers fastd config:
+
+    - | Set ``mode multitap;``
+      | Every peer gets their own interface.
+
+    - | Replace ``interface "foo":`` with ``interface "peer-%k";``
+      | ``%k`` is substituted for a portion of the peers public key.
+
+    - | Set ``offload l2tp yes;``
+      | This tells fastd to use the l2tp kernel module.
+
+    - | Set ``persist interface no;``
+      | This tells fastd to only keep interfaces arround while the connection is active.
+
 Note that in ``multitap`` mode, which is required when using
 L2TP offloading, fastd will create one interface per peer
-on the supernode's side and it is the administrator's
-responsibility to ensure that these interfaces are handled correctly.
-In batman-adv-based setups this involves adding the dynamically created
-interfaces to an batadv interface using fastd's ``on up`` scripts or some
-network configuration daemon like systemd-networkd.
+on the supernode's. This allows offloading the L2TP forwarding into the 
+kernel space. But this also means added copmlexity with regards to handling
+those interfaces.
+
+There are two main options how you can handle this:
+
+    -  create ``on up`` and ``on down`` hooks
```

Comment:
establish/disestablish

Reply:
https://git.darmstadt.ccc.de/ffda/infra/salt/-/commit/d59e6bad271dd6cd23040331043c3d13ce8bce74

https://git.darmstadt.ccc.de/ffda/infra/salt/-/commit/bf8a084f3eb5a592f6ae8711bfd99fc842ef58ce

i don't remeber why you changed it, but we use up and down

Reply:
Wow, was looking at an outdated checkout, my bad.

All context should be here https://github.com/NeoRaider/fastd/issues/26.